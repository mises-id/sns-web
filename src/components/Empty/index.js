/*
 * @Author: lmk
 * @Date: 2021-07-16 14:00:13
 * @LastEditTime: 2021-07-16 14:10:49
 * @LastEditors: lmk
 * @Description: empty page
 */
import empty from '@/images/empty.png'
import React from 'react';
import './index.scss'
import { useTranslation } from 'react-i18next';
const Empty = (props)=>{
  const {t} = useTranslation()
  return <div className="m-text-center emptyBox">
    <img alt="empty" src={empty} className="emptyImg"></img>
    <p className="emptyTxt">{t('empty')}</p>
  </div>
}
export default Empty