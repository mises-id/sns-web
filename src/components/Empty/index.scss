.emptyImg{
  width: 237px;
  height: 130px;
}
.emptyBox{
  margin-top: 100px;
}
.emptyTxt{
  font-size: 17px;
  color: #BBBBBB;
  margin-top: 6px;
}