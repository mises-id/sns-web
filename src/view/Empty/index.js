/*
 * @Author: lmk
 * @Date: 2021-08-18 22:36:01
 * @LastEditTime: 2021-08-18 22:36:18
 * @LastEditors: lmk
 * @Description: empty
 */
import React from 'react';
const Empty = (props)=>{
  return <div></div>
}
export default Empty