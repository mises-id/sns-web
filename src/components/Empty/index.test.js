/*
 * @Author: lmk
 * @Date: 2021-07-15 10:12:21
 * @LastEditTime: 2021-07-16 14:16:20
 * @LastEditors: lmk
 * @Description: Empty test
 */
import React from 'react';
import {shallow} from 'enzyme';
import Empty from '.';
describe('Empty', () => {
  it('should render correctly', () => {
    const component = shallow(<Empty />)
    expect(component).toMatchSnapshot();
  });
});