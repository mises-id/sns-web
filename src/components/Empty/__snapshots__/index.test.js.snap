// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Empty should render correctly 1`] = `
<div
  className="m-text-center emptyBox"
>
  <img
    alt="empty"
    className="emptyImg"
    src="empty.png"
  />
  <p
    className="emptyTxt"
  >
    empty
  </p>
</div>
`;
